packages/go_router/lib/src/route.dart

```diff
@@ -279,3 +317,62 @@ class GoRoute {
   ) =>
       const SizedBox.shrink();
 }
+
+/// A route that displays a UI shell around the matching child route. Builds
```

Comment:
the first paragraph of a doc string should be one brief sentence.

Reply:
62480777c587e92b002e092dbfca7da61cd021db